lib/msal-core/dist/msal.js

```diff
@@ -2577,13 +2587,22 @@ var UserAgentApplication = /** @class */ (function () {
                 }
                 var scope_1 = scopes.join(" ").toLowerCase();
                 var userObject_1 = user ? user : _this.getUser();
-                if (!userObject_1) {
+                var adalIdToken = _this._cacheStorage.getItem(Constants_1.Constants.adalIdToken);
```

Comment:
I agree. When I created this PR, I made the changes for both option 2(don't change getUser()) and option 3(create MSAL object from adal's idToken). I just wanted to demonstrate how the changes will look like if we go with one option or the other. Since we are going ahead with option 3, I don't need this code.